client/src/components/advancedSelectWidget/AdvancedSelect.js

```diff
@@ -1,25 +1,22 @@
 import API from "api"
 import { gql } from "apollo-boost"
-import { renderInputField } from "components/FieldHelper"
-import LinkTo from "components/LinkTo"
+import * as FieldHelper from "components/FieldHelper"
 import UltimatePagination from "components/UltimatePagination"
-import { Field } from "formik"
 import _debounce from "lodash/debounce"
 import _isEmpty from "lodash/isEmpty"
 import _isEqual from "lodash/isEqual"
 import PropTypes from "prop-types"
 import React, { Component } from "react"
-import { Button, Col, Overlay, Popover, Row } from "react-bootstrap"
-import ContainerDimensions from "react-container-dimensions"
-import "./AdvancedMultiSelect.css"
+import { Button, Col, FormControl, InputGroup, Row } from "react-bootstrap"
+import { Popover, PopoverInteractionKind, Position } from "@blueprintjs/core"
 
-const MOBILE_WIDTH = 733
+import "./AdvancedSelect.css"
```

Comment:
Please honour the correct ordering of imports, and add no blank lines between imports.